bar-trender-front-end/src/components/withListLoading.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function WithListLoading(Component) {
+  return function WihLoadingComponent({ isLoading, ...props }) {
+    if (!isLoading) return <Component {...props} />;
+    return (
+      <p style={{ textAlign: 'center', fontSize: '30px' }}>
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7122945)
```suggestion
      <p style={{ textAlign: "center", fontSize: '30px' }}>
```